greedy_approach/krushkals.c

```diff
@@ -0,0 +1,91 @@
+/**
+ * @krushkals.c
+ * @reference:https://www.geeksforgeeks.org/kruskals-minimum-spanning-tree-algorithm-greedy-algo-2/
+ * @details
+ * Kruskal’s algorithm is a greedy algorithm in graph theory that finds a minimum spanning tree for a connected weighted graph.
+ * theory that finds a minimum spanning tree for a connected weighted graph.
+ * It finds a subset of the edges that forms a tree that includes every vertex, * where the total weight of all the edges in the tree is minimized.
+ * @author [Ankit Akash](https://github.com/ankit-akash)
+ */
+ 
+  #include <stdio.h>
+    #include <conio.h>
+    #include <stdlib.h>
+
```

Comment:
Please add a one-line description of what the library/header is for (**see the example below**).

```c
#include <stdio.h>     /// for IO operations
#include <assert,h>    /// for assert
```